robolectric-resources/src/main/java/org/robolectric/res/ResType.java

```diff
@@ -49,6 +56,25 @@ private static TypedResource extractScalarItems(XpathResourceXmlLoader.XmlNode x
     return new TypedResource<>(typedResources, arrayResType, xmlContext);
   }
 
+  private static TypedResource extractTypedItems(XpathResourceXmlLoader.XmlNode xmlNode, ResType arrayResType, XmlLoader.XmlContext xmlContext) {
+    final List<TypedResource> items = new ArrayList<>();
+    for (XpathResourceXmlLoader.XmlNode item : xmlNode.selectElements("item")) {
+      final String itemString = item.getTextContent();
+      ResType itemResType = inferFromValue(itemString);
+      if (itemResType == ResType.CHAR_SEQUENCE) {
+        if (itemString.startsWith("?")) {
+          itemResType = ResType.STYLE;
```

Comment:
Yes -- TypedArrays only save (and therefore, only allow access to) the resource id of the themed attribute, which means this should work for all attributes.